### PR TITLE
Add cli option to add suspend modifier to controller methods

### LINKED ISSUE #112: Support generating controller methods with `suspend` (for spring-webflux)
spring-webflux supports Kotlin `suspend` methods in controllers. Seems like this would be simple for fabrikt to support, though it would have to be optional.

The `suspend` modifier should be the only change that's needed for webflux support, otherwise the controllers generated by fabrikt look pretty much identical to the ones we use right now generated by openapi-generator.

### PR DESCRIPTION
**Pull request by averabaq**
Add a `suspend_modifier` cli-option to add the `suspend` modifier to controller methods

Closes #112

### CODE REVIEW

**Comment by cjbooms** [comment]
Looks good to me. Maybe wait for a thumb from @Gama11

**Comment by cjbooms** [comment]
👍

**Review by cjbooms — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -123,29 +123,31 @@ data class ConcreteImplTwo(
...
- |                        |   `MICRONAUT_REFLECTION` - This option adds @ReflectiveAccess to the generated models. Requires dependency "'io.micronaut:micronaut-core:+"
- |   `--output-directory` | Allows the generation dir to be overridden. Defaults to current dir
- |   `--targets`          | Targets are the parts of the application that you want to be generated.
- |                        | CHOOSE ANY OF:
- |                        |   `HTTP_MODELS` - Jackson annotated data classes to represent the schema objects defined in the input.
- |                        |   `CONTROLLERS` - Spring annotated HTTP controllers for each of the endpoints defined in the input.
- |                        |   `CLIENT` - Simple http rest client.
- |                        |   `QUARKUS_REFLECTION_CONFIG` - This options generates the reflection-config.json file for quarkus integration projects
-
+| Parameter                  | Description
```
[the remark this answers is not in the document]
Comment (averabaq) [reply README.md:126]: The thing is that the new `--http-controller-opts` option increased the column spacing and moved everything around, that's the reason of the big delta here 🙂

**Comment by averabaq** [comment]
@Gama11 would this meet your requirement?

**Comment by Gama11** [comment]
Yes, looks good. Thanks!

**Pull request merged**